accesstoken2/src/services/config-service.ts

```diff
@@ -0,0 +1,96 @@
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+
+const SESSION_TABLE_NAME_KEY = "SessionTableName";
+const DEFAULT_AUTHORIZATION_CODE_TTL_IN_SECS = 600;
+const PARAMETER_PREFIX = process.env.AWS_STACK_NAME || "";
+const ACCESS_TOKEN_TTL = process.env.ACCESS_TOKEN_TTL_IN_SECS || 3600;
+
+export class ConfigService {
+    private readonly authorizationCodeTtlInMillis: number;
+    private readonly configEntries: Map<string, string>;
+
+    constructor(private ssmClient: SSMClient) {
+        const envAuthCodeTtl = parseInt(process.env.AUTHORIZATION_CODE_TTL || "", 10);
+        this.authorizationCodeTtlInMillis =
+            (Number.isInteger(envAuthCodeTtl) ? envAuthCodeTtl : DEFAULT_AUTHORIZATION_CODE_TTL_IN_SECS) * 1000;
+        this.configEntries = new Map<string, string>();
+    }
+
+    public init(): Promise<string[]> {
+        const REDIRECT_URI = `clients/ipv-core-stub/jwtAuthentication/redirectUri`;
```

Comment:
We have one of the constant keys outside the init `SESSION_TABLE_NAME_KEY` and one inside. We should only use one pattern. I'd suggest setting this value up on line 2/3